### PR TITLE
[UNDERTOW-1637] Redirect Form auth on root context without trailing s…

### PR DESCRIPTION
**Pull request by spyrkob**
…lash

Issue: https://issues.redhat.com/browse/UNDERTOW-1637

@fl4via could you please review? Alternatively the redirect could be handled in https://github.com/undertow-io/undertow/blob/master/servlet/src/main/java/io/undertow/servlet/handlers/ServletInitialHandler.java#L140, but doing that would undo https://issues.redhat.com/browse/UNDERTOW-1578 for requests to root context

### CODE REVIEW

**Comment by spyrkob** [comment]
retest this please

File: servlet/src/test/java/io/undertow/servlet/test/security/form/FormAuthenticationRootContextRedirectTestCase.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
+/**
```
Comment (fl4via) [inline servlet/src/test/java/io/undertow/servlet/test/security/form/FormAuthenticationRootContextRedirectTestCase.java]: I think we are missing the first block in the license header? Is there a reason for this or was it mistakenly removed?

Comment (spyrkob) [reply servlet/src/test/java/io/undertow/servlet/test/security/form/FormAuthenticationRootContextRedirectTestCase.java]: Looks like I had wrong template in the IDE, I'll change that

**Comment by spyrkob** [comment]
@fl4via the license is updated now. Apart from that is  the PR looking OK?

**Pull request merged**